ui/src/app/settings/components/project-sync-windows-edit/project-sync-windows-edit.tsx

```diff
@@ -14,7 +14,7 @@ function helpTip(text: string) {
         <Tooltip content={text}>
             <span style={{fontSize: 'smaller'}}>
                 {' '}
-                <i className='fa fa-question-circle' />
```

Comment:
I think this should still be a question icon. This is a tooltip with a help text, so it is ok to use question mark.